src/main/java/org/sonarsource/sonarlint/ls/connected/ProjectBindingManager.java

```diff
@@ -167,14 +170,17 @@ private ConnectedSonarLintEngine createConnectedEngineAndUpdateIfNeeded(String s
       LOG.error("Error starting connected SonarLint engine for '" + serverId + "'", e);
       return null;
     }
-
+    List<String> failedServerIds = new ArrayList<>();
     try {
       GlobalStorageStatus globalStorageStatus = engine.getGlobalStorageStatus();
       if (forceUpdate || globalStorageStatus == null || globalStorageStatus.isStale() || engine.getState() != State.UPDATED) {
-        updateGlobalStorageAndLogResults(serverConfiguration, engine);
+        updateGlobalStorageAndLogResults(serverConfiguration, engine, failedServerIds, serverId);
       }
     } catch (Exception e) {
       LOG.error("Error updating storage of the connected SonarLint engine '" + serverId + "'", e);
+      if (!failedServerIds.isEmpty()) {
+        client.showMessage(new MessageParams(MessageType.Error, "Binding update failed for the server: " + serverId));
```

Comment:
Maybe add "Look to the SonarLint output for details"?

Reply:
Done.